client/helpers/get-key-value-pairs.js

```diff
@@ -53,6 +53,17 @@ const getKeyValuePairs = event => {
           routeLink: value.routeLink,
           value: value.text,
         });
+      } else if (key === 'expirationTime') {
+        const val = timestampToDate(value);
+
+        if (!val || !val.isValid()) {
+          return;
+        }
+
+        kvps.push({
+          key,
+          value: val.year() === 1969 ? '' : val.format('lll'), // 1969 corresponds to no expiration timeout
```

Comment:
We should probably check to make sure `val` and `val.year` exist before calling `val.year()`, otherwise, things could blow up at runtime. 💥 The same probably goes for `val.format`.

As far as I can tell, we don't have TypeScript looking out for us. So, we're on our own to protect ourselves from the edge case where `undefined` creeps in.

I don't think it really matters here, but I might break out this logic into a well-named helper function.

```
const NO_EXPIRATION_TIMEOUT = 1969;

const hasNoExpirationTimeout = val => {
  // Maybe we should handle these edge cases differently?
  if (!val) return;
  if (!val.year) return;
  if (!val.format) return;
  if (val.year() === NO_EXPIRATION_TIMEOUT) return '';

  return val.format('lll');
};
```

Reply:
done +